src/libs/runtime-utils.js

```diff
@@ -272,7 +272,7 @@ export const appIsSettingUp = app => {
   return app && (app.status === 'PROVISIONING' || app.status === 'PRECREATING')
 }
 
-export const getCurrentPersistentDisk = (appType, apps, appDataDisks) => {
+export const getCurrentPersistentDisk = (appType, apps, appDataDisks, workspaceName) => {
```

Comment:
Please add unit test coverage for this argument.

Reply:
will do